airflow/providers/microsoft/mssql/hooks/mssql.py

```diff
@@ -22,10 +22,13 @@
 from typing import Any
 
 import pymssql
-from methodtools import lru_cache
 from pymssql import Connection as PymssqlConnection
 
-from airflow.providers.common.sql.hooks.sql import DbApiHook, fetch_all_handler
+from airflow.providers.common.sql.dialects.mssql import MsSqlDialect
+from airflow.providers.common.sql.hooks.sql import DbApiHook
+
+if TYPE_CHECKING:
```

Comment:
you have missing import of TYPE_CHECKING.
add to this file `from typing import TYPE_CHECKING`

Reply:
Thanks @romsharon98 👍

Reply:
How could I have missed this didn't see it an PyCharm didn't complain about it either, thx again @romsharon98